### PR TITLE
Save for later show more

### PR DESCRIPTION
**Pull request by sndrs**
after a few battles trying something nicer, this just formats the button and tidies the items a little for saved for later. if the test is successful, a deeper refactor will probably be needed to align ID and facia a bit, now that content is shared.

this doesn't improve layout, which will be a separate PR.

screen shot 2015-06-09 at 16 46 50

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/loyalty/save-for-later.js [hunk 1]
```diff
@@ -96,19 +96,6 @@ define([
         });
     };
 
-    SaveForLater.prototype.renderSaveThisArticleLink = function (deferToClick, url, state) {
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/loyalty/save-for-later.js]: this was a dupe

File: static/src/stylesheets/module/identity/_identity.scss [hunk 2]
```diff
@@ -422,18 +414,32 @@
     @include fs-bodyCopy(1);
 }
 
+.saved-content__meta {
+    padding-top: $gs-baseline;
+    text-align: right;
+
+    .button {
```
Comment (uplne) [inline static/src/stylesheets/module/identity/_identity.scss]: I suppose this is to get around some specificity issues. Could we rather use '.saved-content__button'? So the changes will be visually coupled only within the `saved-content` object.

Comment (sndrs) [inline static/src/stylesheets/module/identity/_identity.scss]: yes - good catch

**Comment by uplne** [comment]
:+1:

**Pull request merged**